iron-list.html

```diff
@@ -1390,11 +1390,14 @@
 
       forceUpdate = forceUpdate || this._scrollHeight === 0;
       forceUpdate = forceUpdate || this._scrollPosition >= this._estScrollHeight - this._physicalSize;
+      // Probably a bug that style.height is a string like '100px' and it is being compared to
+      // a number?
       forceUpdate = forceUpdate || this.grid && this.$.items.style.height < this._estScrollHeight;
 
       // Amortize height adjustment, so it won't trigger large repaints too often.
       if (forceUpdate || Math.abs(this._estScrollHeight - this._scrollHeight) >= this._optPhysicalSize) {
         this.$.items.style.height = this._estScrollHeight + 'px';
+        this.updateViewportBoundaries();
```

Comment:
As a person who's told a large number of people "need to properly size the iron-list's parent" myself, I'd personally love to see a repro where that actively wasn't the case, if for nothing else than to add some empathy to my response next time. The text in your "in short" makes me think there's a `this.notifyResize()` missing, but I'd love to know better when making suggestions to others on that point, too!